packages/server/src/api/routes/tests/datasource.spec.ts

```diff
@@ -165,7 +165,8 @@ describe("/datasources", () => {
 })
 
 const descriptions = datasourceDescribe({
-  exclude: [DatabaseName.MONGODB, DatabaseName.SQS],
+  plus: true,
```

Comment:
You'll notice a lot of changes like this one dotted around. I didn't want adding a new datasource to the list of datasources to mean it gets added to all of the places that specify just an `exclude` list on `datasourceDescribe`. I noticed all of the places that do this are doing tests that should pass for all DS+, so I've added this parameter to constrain those.